pystiche_papers/johnson_alahi_li_2016/_loss.py

```diff
@@ -38,6 +38,25 @@ def content_loss(
     layer: str = "relu2_2",
     score_weight: Optional[float] = None,
 ) -> ops.FeatureReconstructionOperator:
+    r"""Content_loss from "Perceptual Losses for Real-Time Style Transfer and
+    Super-Resolution" by Johnson, Alahi, and Li (original authors) :cite:`JAL2016`.
+
+    Args:
+        impl_params: If ``True``, use the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper. For details see FIXME.
+        instance_norm: If ``True``, use :class:`~torch.nn.InstanceNorm2d` rather than
+            :class:`~torch.nn.BatchNorm2d` as described in the paper.
+        style: Optional style for selecting the correct optimization parameters for the reproduction.
```

Comment:
1. What do you mean by correct? 
2. I would swap "reproduction" with "replication"